pkg/move/cmd.go

```diff
@@ -23,8 +26,20 @@ var (
 	sourceFolder string
 	targetFolder string
 	workFolder   string
+	technology   string
 )
 
+type Manifest struct {
+	Technologies map[string]map[string][]FileEntry `json:"technologies"`
+	Version      string                            `json:"version"`
+}
+
+type FileEntry struct {
+	Path    string `json:"path"`
+	Version string `json:"version"`
+	MD5     string `json:"md5"`
+}
```

Comment:
the type `map[string]map[string][]FileEntry` is a mouthful, but creating a bunch of `structs` for it makes no sense because then you can't use them as a map

but maybe you could do some aliasing, so it reads nicer 
- hopefully it works like I expect it
- the names are not final :D 

```suggestion
type Manifest struct {
	Technologies TechEntries `json:"technologies"`
	Version      string                            `json:"version"`
}

type TechEntries map[string]ArchEntries

type ArchEntries map[string][]FileEntry

type FileEntry struct {
	Path    string `json:"path"`
	Version string `json:"version"`
	MD5     string `json:"md5"`
}
```

Reply:
done https://github.com/Dynatrace/dynatrace-bootstrapper/pull/4/commits/3bc29f55aae9afcb84fa1845556cbb335785cdf1